cli/schemas/interchain-tokenlist.ts

```diff
@@ -13,6 +13,10 @@ export const remoteToken = z.object({
 export const interchainToken = z.object({
   tokenId: hash,
   tokenAddress: address,
+  tokenManagerAddress: address,
+  deployerAddress: address,
+  originalMinterAddress: address,
+  tokenManagerType: z.string(),
```

Comment:
```suggestion
  tokenManager: address,
  deployer: address,
  originalMinter: address,
  tokenManagerType: z.string(),
```
address suffix is redundant